src/modules/folder/folder.controller.spec.ts

```diff
@@ -47,4 +81,187 @@ describe('FolderController', () => {
       );
     });
   });
+
+  describe('get folder content', () => {
+    it('When get folder subfiles are requested by folder uuid, then the child files are returned', async () => {
+      const expectedSubfiles = [
+        newFile({ attributes: { id: 1, folderUuid: folder.uuid } }),
+        newFile({ attributes: { id: 2, folderUuid: folder.uuid } }),
+        newFile({ attributes: { id: 3, folderUuid: folder.uuid } }),
+      ];
+      jest.spyOn(fileUseCases, 'getFiles').mockResolvedValue(expectedSubfiles);
+
+      const result = await folderController.getFolderContentFiles(
+        userMocked,
+        folder.uuid,
+        50,
+        0,
+        'id',
+        'ASC',
+      );
+      expect(result).toEqual({ files: expectedSubfiles });
+    });
+
+    it('When get folder subfolders are requested by folder uuid, then the child folders are returned', async () => {
+      const expectedSubfolders = [
+        newFolder({ attributes: { id: 1, parentUuid: folder.uuid } }),
+        newFolder({ attributes: { id: 2, parentUuid: folder.uuid } }),
+        newFolder({ attributes: { id: 3, parentUuid: folder.uuid } }),
+      ];
+      const mappedSubfolders = expectedSubfolders.map((f) => {
+        let folderStatus: FileStatus;
+        if (f.removed) {
+          folderStatus = FileStatus.DELETED;
+        } else if (f.deleted) {
+          folderStatus = FileStatus.TRASHED;
+        } else {
+          folderStatus = FileStatus.EXISTS;
+        }
+        return { ...f, status: folderStatus };
+      });
+
+      jest
+        .spyOn(folderUseCases, 'getFolders')
+        .mockResolvedValue(expectedSubfolders);
+
+      const result = await folderController.getFolderContentFolders(
+        userMocked,
+        folder.uuid,
+        50,
+        0,
+        'id',
+        'ASC',
+      );
+
+      expect(result).toEqual({ folders: mappedSubfolders });
+    });
+
+    it('When get folder subfiles are requested by invalid params, then it should throw an error', async () => {
+      try {
+        await folderController.getFolderContentFiles(
+          userMocked,
+          'invalidUUID',
+          50,
+          0,
+          'id',
+          'ASC',
+        );
+        expect(true).toBeFalsy; // getFolderContentFiles should throw an error
+      } catch (err) {
+        expect((err as Error).message).toEqual('Invalid UUID provided');
+      }
+
+      try {
+        await folderController.getFolderContentFiles(
+          userMocked,
+          folder.uuid,
+          0,
+          0,
+          'id',
+          'ASC',
+        );
+        expect(true).toBeFalsy; // getFolderContentFiles should throw an error
```

Comment:
The comment is not adding something difficult to understand, the code is semantic enough to understand that "the test expects the function to throw an error with a specific message. 

By the way, the most maintainable solution to match errors is using `instanceof` as if the limit minimum or maximum numbers change, the message is prone to change, so this test would need to be updated. So, it would be better to find something like: 

`expect(err instanceof TheErrorIWantToMatch).isTruthy();`